src/pm/hydra/mpiexec/pmiserv_publish.c

```diff
@@ -118,7 +118,7 @@ HYD_status HYD_pmiserv_lookup(struct HYD_proxy *proxy, int pid, int pgid, struct
 
   fn_exit:
     if (value)
-        MPL_free(value);
+        MPL_free((void *) value);
```

Comment:
Add comment explain the purpose is to cast away the `const`

Reply:
Done.